django/econsensus/publicweb/forms.py

```diff
@@ -72,3 +73,12 @@ class FilterForm(forms.Form):
                          initial=EXTRA_CHOICE[0],
                          required=False,
                          widget = forms.Select(attrs={'onchange':'this.form.submit()'}))
+
+class ChangeOwnerForm(forms.ModelForm):
+    class Meta:
+        model = OrganizationOwner
+        fields = ['organization_user']
+    def __init__(self, *args, **kwargs):
+        currentOrgPk = kwargs.pop("currentOrgPk")
```

Comment:
it's good practice in python to use "snake case" current_org_pk for variable & method names.